os/emscripten/README.md

```diff
@@ -13,19 +13,20 @@ Build the docker image:
   docker build -t emsdk-lzma .
 ```
 
+From the repository root
 Build the host tools first:
 ```
   mkdir build-host
-  docker run -it --rm -v $(pwd):$(pwd) -u $(id -u):$(id -g) --workdir $(pwd)/build-host emsdk-lzma cmake .. -DOPTION_TOOLS_ONLY=ON
-  docker run -it --rm -v $(pwd):$(pwd) -u $(id -u):$(id -g) --workdir $(pwd)/build-host emsdk-lzma make -j5 tools
+  docker run -it --rm -v $(pwd):$(pwd) --workdir $(pwd)/build-host emsdk-lzma cmake .. -DOPTION_TOOLS_ONLY=ON
```

Comment:
You mention it "breaks" on machines you tried, without specifying what breaks. You really want to use `-u`, as it makes sure the files in the build folder are not owned by `root`, but by your own user. That makes it much more practical to work with. So I would prefer if those statements stay in there, as these are what I copy/paste when working with emscripten :)

Reply:
The processes in the docker container were not able to write to the host FS
I tried it on both wsl using podman and a ubuntu vm with docker with the same result
the files without the -u were created as the same user that ran the container

Reply:
Tried it on several machines, I cannot replicate your outcome. And it really shouldn't work like you describe, at least not for docker. Podman is slightly different if you run it in rootless mode, but normal docker always runs everything under `root`. And as such, the files inside the build folder are also assigned to user `root`, which is a terrible user-experience if you don't understand what just happened :)

As such, I would very much prefer that we keep the `-u` in there. If it doesn't work for you, it is easy to remove. But the reverse is more complicated.

Tnx :)

Reply:
Interesting, It might be because I ran docker under rootless mode or something different in my environment, I will remove the change from this PR